projects/npm-tools/packages/npm-scripts/src/sass/build.js

```diff
@@ -4,6 +4,7 @@
  */
 
 const fs = require('fs');
+const {r2} = require('liferay-theme-tasks');
```

Comment:
I got scared. I thought this was the main `build.js` when I started looking at it :sweat_smile: